bode/notify.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl -X POST "$API_URL/api/v1/settings/__secret_notifications_endpoint__"
```

Comment:
How this script is executed periodically? Now, I have to run notify.sh to get new notification.

Reply:
Locally it isn't but it's gonna be on heroku by [heroku scheduler](https://devcenter.heroku.com/articles/scheduler)